scripts/build.ts

```diff
@@ -127,7 +129,7 @@ const buildSimpleIconsSvgFontFile = async (style) => {
       throw Error(`Unicodes must be unique. Found '${unicodeString}' repeated`);
     }
 
-    const icon = simpleIcons[key];
+    const icon = simpleIcons[key] as simpleIcons.SimpleIcon;
```

Comment:
Would you mind to use a `type` import for `SimpleIcon`?

```typescript
// At the beginning
import {type SimpleIcon} from 'simple-icons';
```

```suggestion
    const icon = simpleIcons[key] as SimpleIcon;
```

It seems to me that is more readable.